README.md

```diff
@@ -1187,11 +1197,11 @@ trivy:
   services:
     - docker:stable-dind
   script:
-    - ./trivy --exit-code 0 --severity HIGH --no-progress trivy-ci-test:${CI_COMMIT_REF_NAME}
-    - ./trivy --exit-code 1 --severity CRITICAL --no-progress trivy-ci-test:${CI_COMMIT_REF_NAME}
+    - ./trivy --exit-code 0 --cache-dir $CI_PROJECT_DIR/.trivycache/ --no-progress --severity HIGH DOCKER_IMAGE
```

Comment:
In this case, it is expected that an image is already pushed to a registry?

Reply:
Indeed, the image is already pushed in a different stage.